openmls/src/messages/mod.rs

```diff
@@ -283,16 +283,15 @@ pub(crate) enum PathSecretError {
 
 /// GroupSecrets
 ///
-/// > 11.2.2. Welcoming New Members
-///
-/// ```text
+/// ```c
+/// // draft-ietf-mls-protocol-17
 /// struct {
-///   opaque joiner_secret<1..255>;
+///   opaque joiner_secret<V>;
 ///   optional<PathSecret> path_secret;
-///   optional<PreSharedKeys> psks;
+///   PreSharedKeyID psks<V>;
 /// } GroupSecrets;
 /// ```
-#[derive(TlsDeserialize, TlsSize)]
+#[derive(Debug, TlsDeserialize, TlsSize)]
 pub(crate) struct GroupSecrets {
     pub(crate) joiner_secret: JoinerSecret,
```

Comment:
This is a wrapper of `Secret` and thus would be printed as "<redacted>" during non`crypto-debug` builds.